spec/ibandit/pseudo_iban_splitter_spec.rb

```diff
@@ -15,6 +15,15 @@
       its([:account_number]) { is_expected.to eq("0105723") }
     end
 
+    context "for an australian pseudo-IBAN" do
+      let(:pseudo_iban) { "AUZZ123456123456789" }
+
+      its([:country_code]) { is_expected.to eq("AU") }
+      its([:bank_code]) { is_expected.to be_nil }
```

Comment:
Why don't we want to extract bank_code too?

Reply:
All we know is the bsb number, which identifies a branch. The first 2 or 3 characters represent the branch, but I am not sure how we can pull out the bank_code from the bsb as I am unsure how we can detect whether we want to pull out the first 2 or 3 chars. For reference, take a look at the `Number` and `Code` columns in the table here: https://www.thebsbnumbers.com/

That said, I don't think the second commit of this PR does the right thing.